articles/install-app-store-apps.md

```diff
@@ -4,7 +4,7 @@ _Available in Fleet Premium_
 
 In Fleet, you can install Apple App Store apps on your macOS, iOS, and iPadOS hosts. To do this, you must first [turn on Apple MDM](https://fleetdm.com/guides/apple-mdm-setup#turn-on-apple-mdm) and Apple's [Volume Purchasing Program (VPP)](https://fleetdm.com/guides/apple-mdm-setup#volume-purchasing-program-vpp).
 
-You can also manage which Google Play Store apps are available for self-serivce in your end user's Android work profiles.
+You can also manage which Google Play Store apps are available for self-serivce in your end user's Android work profiles. Paid Google Play Store apps are [not supported in Android MDM](https://www.androidenterprise.community/discussions/conversations/distributing-paid-apps/653).
```

Comment:
```suggestion
You can also manage which Google Play Store apps are available for self-serivce in your end user's Android work profiles. Google only allows free Google Play Store apps. [Paid apps aren't supported](https://www.androidenterprise.community/discussions/conversations/distributing-paid-apps/653).
```

Nice piece of documentation work!

I tweaked the language so folks know it's not a Fleet limitation. It's enforced by Google. Is that right?